src/index.js

```diff
@@ -1 +1,88 @@
-// Lets do this!
+import { toRefs, h, computed, markRaw, watch, getCurrentInstance } from 'vue'
+import { useForm, useField } from 'vee-validate';
+
+/**
+ * For a Schema, find the elements in each of the rows and remap the element with the given function
+ * @param {Array} schema
+ * @param {Function} fn
+ *
+ * @returns {Array}
+ */
+export const mapElementsInSchema = (schema, fn) => schema.map(row => row.map(el => fn(el)))
+
+export default function VeeValidatePluginFactory () {
+  return function VeeValidatePlugin(baseReturns) {
+    // Take the parsed schema from SchemaForm setup returns
+    const { parsedSchema, formBinds } = baseReturns
+  
+  
+    // Get additional properties not defined on the `SchemaForm` derivatives
+    const { attrs: formAttrs } = getCurrentInstance();
+    // Create a form context and inject the validation schema if needed
+    const { handleSubmit } = useForm({
+      validationSchema: formAttrs['validation-schema'] || formAttrs['validationSchema']
+    });
```

Comment:
`SchemaFormFactory` returns after the plugins are applied a `SchemaForm`, which in return creates a wrapping `form` element. The top level `SchemaForm` will always be a `form`, as it is defined by a `component :is`.

```
<component
    :is="!hasParentSchema ? 'form' : 'div'"
...
```

I'm not sure what you're trying to do here since I'm not very familiar with your API, but let's chat about this bit

Reply:
Okay, so vee-validate supports 2 levels of validation. Either on field basis, so like this:

```js
const schema = {
  firstName: {
    validations: yup.string().required()
  }
};
```

Or on the form level, so you can separate your validations from the schema definition:

```js
const validations = yup.object().shape({
  firstName: yup.string().required()
});

const schema = {
  // ...
};
```

What I am doing there is a hack, I'm providing a way for formvulate users to pass in the form-level validation schema, using `attrs` present on the schema form component.

So it looks like this when used:

```vue
<SchemaFormWithPlugin
  :schema="schema"
  :validation-schema="validations"
  v-model="formData"
/>
```

This might not be desirable as it would allow plugins to throw stuff on the `attrs`, so let's check if this can be something official or not.

Reply:
I see what you mean. Im not immediately worried about it being throw in the attrs, and the idea is to sort of keep FVL agnostic to validation - thats the point of having these amazing plugins :)

So yeah lets keep it like this for now, and revisit if/when needed